hail/python/hailtop/hailctl/__main__.py

```diff
@@ -6,6 +6,7 @@
 import time
 
 from hailtop import hailctl
```

Comment:
failing the linter since this import is now unused

Reply:
Fixed in: a94d96c